frontend/default/routes.py

```diff
@@ -1,8 +1,8 @@
-from flask import Blueprint, current_app
-from flask import render_template
 from config import Config
-
-from models.data import get_round_data
+from flask import Blueprint
+from flask import current_app
+from flask import render_template
+from models.data import get_round_data_fail_gracefully
```

Comment:
Worth moving that graceful failure method into utils rather than duplicating in different services?

Reply:
Do in future ticket